### PR TITLE
Fix $modal method on root admin.vue

### LINKED ISSUE #722: Member modal display is broken
## Detailed Description
The modal displaying the member is broken:

screenshot-www data dev 2017-01-18 20-22-27

## Context
Needed to display current members roles, update it...

## Possible Implementation
Need a fix

## Your Environment
Version 1.0.0.dev1215

### PR DESCRIPTION
**Pull request by noirbizarre**
Fix `$modal` method making use of `propsData` in `admin.vue` and  use only properties in component (fix #722)
There is a lot of ES6 changes noise on members and members-modal.

### CODE REVIEW

File: js/components/organization/members.vue [hunk 1]
```diff
@@ -145,87 +145,84 @@
...
+            avatar_placeholder: placeholders.user,
             placeholder: this._('Type an user name'),
             requests: new Requests(),
             adding: false,
             validating: false,
         };
     },
     events: {
-        'completer:item-add': function(user_id, $item) {
+        'completer:item-add'(user_id, $item) {
```
Comment (davidbgk) [inline js/components/organization/members.vue]: Wow, are you sure of that syntax or is it a failed global find/replace?

Comment (noirbizarre) [inline js/components/organization/members.vue]: A failed find & replace :/

Comment (davidbgk) [inline js/components/organization/members.vue]: That's why reviews are useful ;) 👍

Comment (noirbizarre) [inline js/components/organization/members.vue]: I fixed it because it's weird but it seems to work with this syntax 😱

**Pull request merged**